app/helpers/admin/analytics_helper.rb

```diff
@@ -2,7 +2,7 @@ module Admin
   module AnalyticsHelper
     def track_analytics_data_on_load(title)
       {
-        event_name: "page_view",
+        event: "page_view",
```

Comment:
I wasn't 100% sure if this was supposed to be "event" or "event_name", but all other GA4 events used "event" so I assumed it probably was

Reply:
Probably worth double checking that this still works once it goes through